### PR TITLE
[COLOR] changed color styles to CSS for more files 

### PR DESCRIPTION
**Pull request by ahwnsals1234**

### CODE REVIEW

**Review by jsenning — DISMISSED** [review]
These look good to me.

**Review by Centerville1 — CHANGES_REQUESTED** [review]
lgtm so far!  We'll wait to merge this until we get our main PR merged to develop, so marking it for changes to prevent accidental merging.

**Comment by ahwnsals1234** [comment]
I deleted the OfflineBanner file from 'src/components/OfflineBanner' because considering that it was only being called from the file itself, I assumed it is obsolete.

File: src/components/EventList/components/EventItem/EventItem.module.scss
Comment (Centerville1) [inline src/components/EventList/components/EventItem/EventItem.module.scss:19]: This should probably be neutral light, since it's a background color, and not text(I assume)

File: src/components/Profile/components/VictoryPromise/components/GraphDisplay.tsx
Comment (Centerville1) [inline src/components/Profile/components/VictoryPromise/components/GraphDisplay.tsx:5]: Since this color is used in two places, it might be better to save this color to a variable in this file and then use that, up to you though.

File: src/components/Profile/components/VictoryPromise/components/VictoryPromiseIcons.tsx
Comment (Centerville1) [inline src/components/Profile/components/VictoryPromise/components/VictoryPromiseIcons.tsx:26]: Same thing here with the reused colors, if might be better to save the reused ones to a variable.

Also, just to double check, 'var(--mui-...)' inside quotes didn't work here, right?

File: src/components/Profile/components/VictoryPromise/components/VictoryPromiseIcons.tsx
Comment (Centerville1) [inline src/components/Profile/components/VictoryPromise/components/VictoryPromiseIcons.tsx:46]: I'm not convinced this is the same, since it was doing a typeof call before.  Could you tell what this was doing previously?

**Review by Centerville1 — CHANGES_REQUESTED** [review]
Looks pretty good to me!  Just a couple changes and questions I had.

**Review by Centerville1 — COMMENTED** [review]
lpgtm imho tbh, I'll approve this once we get our main branch merged to develop

**Review by Centerville1 — APPROVED** [review]
Good to go

**Review by Centerville1 — APPROVED** [review]

**Pull request merged**
